src/python/botan.py

```diff
@@ -837,7 +847,7 @@ def test_certs():
         print("Serial number:     %s" % hex_encode(cert.serial_number()))
         print("Authority Key ID:  %s" % hex_encode(cert.authority_key_id()))
         print("Subject   Key ID:  %s" % hex_encode(cert.subject_key_id()))
-        print("Public key bits:\n%s\n" % hex_encode(cert.subject_public_key_bits()))
+        print("Public key bits:\n%s\n" % b2a_base64(cert.subject_public_key_bits()))
```

Comment:
Python 3 team has patched their documentation, removing the wrong statement about length of data:

http://bugs.python.org/file40878/issue25495.patch